engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java

```diff
@@ -681,4 +682,11 @@ ConstraintDeclarationException getInconsistentValueUnwrappingConfigurationBetwee
 
 	@Message(id = 199, value = "Cannot extract value for type %1$s with more than one type parameters.")
 	ValidationException getUnableToExtractValueForTypeWithMultipleTypeParametersException(Type type);
+
+	@LogMessage(level = INFO)
```

Comment:
It's in line with the others. See https://github.com/hibernate/hibernate-validator/blob/master/engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java#L80 for instance.